packages/botfuel-dialog/tests/adapters/messenger-adapter.test.js

```diff
@@ -87,3 +95,126 @@ describe('MessengerAdapter', () => {
     expect(extended).toHaveProperty('payload.value', 'message');
   });
 });
+
+describe('MessengerAdapter request handling', () => {
+  test('should handle a request from messenger', async () => {
+    // Data definitions
+
+    const profile = {
+      firstName: 'Sample',
+      lastName: 'User',
+    };
+    const userMessageUuid = '254a66a3-9282-4e4f-bd86-e18a376d5bc4';
```

Comment:
where do those uuids come from?
if they are random and do not affect tests, maybe we could use something like `uuidv4()`?